docs/src/pages/style/color/color.md

```diff
@@ -38,12 +38,12 @@ const accent2 = purple.A200; // #E040FB (alternative method)
 
 ## Color tool
 
-The Material Design team has built an awesome palette configuration tool: [material.io/tools/color](https://material.io/tools/color/).
-It will help you create a color palette for your UI, as well as measure the accessibility level of any color combination.
 
-<a href="https://material.io/tools/color/#!/?view.left=0&view.right=0&primary.color=3F51B5&secondary.color=F44336">
```

Comment:
Because this tool effectively replaces it. We're linking to the general Material Design color section rather than their color tool. We don't need a picture of theirs alongside ours.

I did wonder if I should add something to display all the generated dark and light colors along those lines.

Reply:
> Because this tool effectively replaces it

I don't think that the new tool answer the same problems. I don't see how it can be a replacement. Let me explain what I see.

tools/color provides:
1. An **official** theme color picker
2. A tool to help picking **accessible** colors

the new demo provides:
1. A way for users to see how the palette **impact** all the components: solve #2009

Vuetify has been trying to replace the official color picker, I don't think that it execute (way to much effort for the outcome, they would have been better off showing how to use the official pickers): https://vuetifyjs.com/en/theme-generator

Reply:
I prefer to keep developers on the docs as much as possible. Our jobs as framework developers is to bring all the good important stuff to the front. As this is a visual demo, I find it has a lot of value. This is why we are rebuilding ours :)

Reply:
> but I'd love some analytics to see if it gets used!

Good point, we can add one Google Analtytics event.

Reply:
> Users can always populate the theme from the official color picker if they choose, and we can keep an explanation of that.

As long as we have that, I'm happy!